tests/test_jats.py

```diff
@@ -110,15 +111,35 @@ def test_jats_cite_context(self):
         for f in filenames:
             test_infile = os.path.join(self.inputdir, f + ".xml")
             test_outfile = os.path.join(self.outputdir, f + ".json")
+            test_outfile_tags = os.path.join(self.outputdir, f + "_tags.json")
             parser = jats.JATSParser()
 
             with open(test_infile, "rb") as fp:
                 input_data = fp.read()
 
+            # Test output as strings
             with open(test_outfile, "rb") as fp:
                 output_text = fp.read()
                 output_data = json.loads(output_text)
-
             cite_context = parser.citation_context(input_data)
-
             self.assertEqual(cite_context, output_data)
+
+            # Test output as BeautifulSoup tags
+            with open(test_outfile_tags, "rb") as fp:
+                output_text = fp.read()
+                output_data_tags = json.loads(output_text)
+            cite_context = parser.citation_context(input_data,num_char=1, text_output=False)
+            # json.loads complained about loading a list of beautifulsoup objects
+            # so the lists are quoted in the test_outfile_tags_file.
+            # This also required some manipulation of the citation_context
+            # output to be equivalent strings.
+            # I imagine there is a better way...
```

Comment:
Add the input and output files to the PR and I'll take a look

Reply:
Pushed the `test_outfile_tags_file`.